dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -1,16 +1,10 @@
 {% macro athena__drop_relation(relation) -%}
-  {% set rel_type = adapter.get_table_type(relation.schema, relation.table) %}
+  {% set rel_type = adapter.get_table_type(relation.schema, relation.table).value %}
   {%- if rel_type is not none %}
     {%- if rel_type == 'table' %}
```

Comment:
see this comment: https://github.com/dbt-athena/dbt-athena/pull/249/files#r1177360995 
could you please add also 
```
{%- if rel_type == 'table' or rel_type='iceberg_table' %}
```
So pretty much before we were avoiding to delete iceberg table data, but now we should clean that up too :)

Reply:
yes it's necessary, we need to be sure that the relation type is either table or iceberg_table, in those cases we prune the s3 path, otherwise not. So it's needed.

Reply:
Will do. I just saw that calling `.value` on the TableType object when it's `None` is returning an empty value. Therefore, I've added an initial `is not none` check on the object before calling `.value`:

https://github.com/dbt-athena/dbt-athena/blob/62b788200d42e4d55fc3ec3c5006b2d63e800e7c/dbt/include/athena/macros/adapters/relation.sql#L1-L4